core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -25,30 +25,30 @@ oppia.directive('simpleEditorSidebar', [function() {
         'ScrollSyncService', 'QuestionIdService',
         function($scope, EditorModeService, SimpleEditorManagerService,
                  ScrollSyncService, QuestionIdService) {
-      $scope.SUBFIELD_LABELS = [
-        'Multiple choice', 'Correct answer', 'Hints', 'Bridge text'];
-      $scope.setEditorModeToFull = EditorModeService.setModeToFull;
-      $scope.questionList = SimpleEditorManagerService.getQuestionList();
-      $scope.ID_PREFIX = QuestionIdService.SIDEBAR_PREFIX;
-      $scope.getSidebarItemId = function(question, subfieldLabel) {
-        return QuestionIdService.getSidebarItemId(
-          question.getId(), subfieldLabel
-        );
-      };
-      $scope.scrollToField = function(question, subfieldLabel) {
-        ScrollSyncService.scrollTo(
-          QuestionIdService.getSubfieldId(
-            question.getId(), subfieldLabel
-          )
-        );
-      };
-      $scope.scrollToHeader = ScrollSyncService.scrollTo;
-      $scope.scrollToQuestion = function(question) {
-        ScrollSyncService.scrollTo(question.getId());
-      };
-      $scope.$on('SimpleEditorSidebarToggleCollapse', function() {
-        $scope.$apply();
-      });
-    }]
+          $scope.SUBFIELD_LABELS = [
```

Comment:
@andromfins -- can you (or someone else) please take care of all these lint issues in a separate PR that's dedicated just to doing that? I think @himanshu-dixit also commented on them, and we really shouldn't leave this branch in an unlinted state to begin with.